src/main/java/codersafterdark/reskillable/base/configs/json/parsers/CustomItemInfoJson.java

```diff
@@ -0,0 +1,26 @@
+package codersafterdark.reskillable.base.configs.json.parsers;
+
+import codersafterdark.reskillable.api.data.ItemInfo;
+import com.google.gson.*;
+import net.minecraft.item.Item;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraftforge.oredict.OreDictionary;
+
+import java.lang.reflect.Type;
+
+public class CustomItemInfoJson implements JsonDeserializer<ItemInfo> {
+    @Override
+    public ItemInfo deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context) throws JsonParseException {
+        JsonObject obj = json.getAsJsonObject();
+
+        Item item = context.deserialize(obj.get("item"), Item.class);
```

Comment:
What if Item is null? Should probably have some sorta of better validation here.